controllers/controlplane_controller.go

```diff
@@ -67,8 +67,35 @@ func (r *ControlPlaneReconciler) Reconcile(ctx context.Context, req ctrl.Request
 		return ctrl.Result{}, nil // no need to requeue, the update will trigger.
 	}
 
+	debug(log, "looking for existing serviceAccounts for ControlPlane deployment", controlplane)
```

Comment:
nit: while I'm sure we don't get this right _everywhere_ it's common to provide camel casing for API types that would match their `kind` even when referencing the type in a log, e.g.:

```suggestion
	debug(log, "looking for existing ServiceAccounts for ControlPlane Deployment", controlplane)
```